<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QWidget" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#settings{
	background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0 #EEAECA, stop: 1 #94BBE9);
	background-color: #A3BCE6;
	border: 2px solid white;
	border: none;
}
*{
	background-color: transparent;
	color: #fff8ea;
	/*border: 2px solid white;*/
	font-size: 16px;
	font-weight: 700;
	font-family: ZCOOL KuaiLe;
}

/*标题*/
#title_lab{
	font-size: 18px;
	color: #CF467F;
	border: none;
}
/*标题*/

/*设置区域*/
[title]{
	color: #CF467F;
	font-size: 20px;
	font-weight: 800;
	border: none;
}
[content]{
	padding-left: 15px;
	border: none;
}
#save_path_lab, #github_lab, #move_widget{
	border-bottom: 2px solid white;
}
/*设置区域*/

/*cookies文本编辑器格式*/
#cookies_textEdit{
	padding: 5px;
	background-color: transparent;
	color: #fff8ea;
	border: 2px solid white;
	font-size: 18px;
	font-weight: 700;
	font-family: ZCOOL KuaiLe;
}
/*cookies文本编辑器格式*/

/*滚动区域*/
QScrollArea{
	border: 2px solid white;
}
/*滚动区域*/

/**/
[content_btn]:hover{
	margin: 2px;
	background-color: rgba(255, 255, 255, 0.4);
	border-radius: 5px;
}
/**/

#close_btn:hover{
	color: white;
	margin: 2px;
}
#close_btn{
	border-image: url(:/title/tools/resource/bg/icon/title/clear.png);
}
#icon_btn{
	border-image: url(:/title/tools/resource/bg/icon/title/settings_3.png);
}
#github_btn{
	border-image: url(:/link/tools/resource/bg/icon/link/github.png);
}
#path_btn{
	border-image: url(:/link/tools/resource/bg/icon/link/folder.png);
}


/*保存取消按钮样式*/
[save]{
	border: 2px solid white;
	border-radius: 5px;
}
[save]:hover{
	color: #DE76A3;
	background-color: rgba(255, 255, 255, 0.3);
	margin: 2px;
}
[save]:pressed{
	color: white;
}
/*保存取消按钮样式*/

/*垂直滚动条样式*/
QScrollBar::add-line, QScrollBar::sub-line {
    width: 0;  /* 隐藏水平滚动条的箭头 */
    height: 0; /* 隐藏垂直滚动条的箭头 */
}
QScrollBar::add-page, QScrollBar::sub-page {
    background: transparent;
    /*background: rgba(255, 255, 255, 0.5);*/
	border: 0px;
	border-radius: 2px;
	padding-top: 2px;
    padding-bottom: 2px;
}
QScrollBar{
    width: 10px;
    border: 0px;
    border-radius: 5px;
	padding: 2px;
}
QScrollBar::handle{
	width: 4px;
	border-radius: 2px;
	/*background-color: #84ACD9;*/
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #639AD4, stop: 1 #DE76A3);
}
QScrollBar::handle:hover{
	/*background-color: #75ACEB;*/
	background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 #639AD4, stop: 1 #DE76A3);
}
QScrollBar:vertical {
    border: none;          /* 去除滚动条整体边框 */
    background: transparent;  /* 背景透明（可选） */
}
QScrollArea .QWidget{
	border: none;
}
QScrollArea{
	padding-top: 5px;
	padding-bottom: 5px;
}
/*垂直滚动条样式*/</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="MoveWidget" name="move_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QPushButton" name="icon_btn">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="title_lab">
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>288</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="close_btn">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>566</width>
          <height>705</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="save_path_title_lab">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>音乐下载路径：</string>
           </property>
           <property name="title" stdset="0">
            <string>-1</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,0">
           <item>
            <widget class="QLabel" name="save_path_lab">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>C:/</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="content" stdset="0">
              <string>-1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="path_btn">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="content_btn" stdset="0">
              <string>-1</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="cookies_title_lab">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Cookies：</string>
           </property>
           <property name="title" stdset="0">
            <string>-1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="cookies_textEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>200</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'ZCOOL KuaiLe'; font-size:18px; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="material_citation_title_lab">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>素材引用：</string>
           </property>
           <property name="title" stdset="0">
            <string>-1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="citation_lab_1">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>音乐：https://music.163.com/</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="content" stdset="0">
            <string>-1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="citation_lab_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>图标：https://www.iconfont.cn/</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="content" stdset="0">
            <string>-1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="save_path_title_lab_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>第三方模块引用：</string>
           </property>
           <property name="title" stdset="0">
            <string>-1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="citation_lab_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>PySide6：https://pypi.org/project/PySide6/</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="content" stdset="0">
            <string>-1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="citation_lab_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>httpx：https://www.python-httpx.org/</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="content" stdset="0">
            <string>-1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="github_title_lab">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>github</string>
           </property>
           <property name="title" stdset="0">
            <string>-1</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0,0">
           <item>
            <widget class="QLabel" name="github_lab">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>github：https://github.com/chisteryg</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="content" stdset="0">
              <string>-1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="github_btn">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="content_btn" stdset="0">
              <string>-1</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>40</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="save_btn">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
         <property name="save" stdset="0">
          <string>-1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancal_btn">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
         <property name="save" stdset="0">
          <string>-1</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MoveWidget</class>
   <extends>QWidget</extends>
   <header>PySide_UI.ui.tools.base.movewidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>settings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancal_btn</sender>
   <signal>clicked()</signal>
   <receiver>settings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>16</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
